src/NuGet.Core/NuGet.Protocol/Model/IgnoreCaseDictionary.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.Protocol.Model
+{
+    internal class IgnoreCaseDictionary<TValue> : Dictionary<string, TValue>
```

Comment:
A class name that would avoid using the verb "Ignore" in its name would be:

```suggestion
    internal class CaseInsensitiveDictionary<TValue> : Dictionary<string, TValue>
```